msticpy/data/uploaders/splunk_uploader.py

```diff
@@ -252,14 +265,19 @@ def upload_folder(  # type: ignore
                 table_name = path.stem
             self._post_data(
                 data=data,
-                table_name=table_name,
+                sourcetype=table_name,
                 index_name=index_name,
                 host=host,
                 create_index=create_index,
             )
             f_progress.update(1)
             if self._debug is True:
-                print(f"{str(path)} uploaded to {table_name}")
```

Comment:
probably get rid of references to table name in the body of the function - if it isn't actually the table_name, it's bit confusing.